src/sentry/models/integrations/sentry_app_component.py

```diff
@@ -1,10 +1,10 @@
 from django.db import models
 
-from sentry.db.models import FlexibleForeignKey, Model, UUIDField, region_silo_only_model
+from sentry.db.models import FlexibleForeignKey, Model, UUIDField, control_silo_only_model
 from sentry.db.models.fields.jsonfield import JSONField
 
 
-@region_silo_only_model
+@control_silo_only_model
```

Comment:
This makes sense, all of sentryapps, components, and the installations should all be control silo models just like integrations.